examples/reading-files/reading-files.go

```diff
@@ -1,84 +1,87 @@
-// Reading and writing files are basic tasks needed for
-// many Go programs. First we'll look at some examples of
-// reading files.
+// Leggere e scrivere su files è un task basilare
+// necessario ad una marea di programma. Vediamo prima
+// qualche esempio di lettura da file.
 
 package main
 
 import (
-    "bufio"
-    "fmt"
-    "io"
-    "io/ioutil"
-    "os"
+	"bufio"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
 )
 
-// Reading files requires checking most calls for errors.
-// This helper will streamline our error checks below.
+// La lettura da file richiede il controllo degli errori
+// di svariate funzioni. Questo helper ci aiuterà a
+// gestire gli errori che si potrebbero generare.
 func check(e error) {
-    if e != nil {
-        panic(e)
-    }
+	if e != nil {
+		panic(e)
+	}
 }
 
 func main() {
 
-    // Perhaps the most basic file reading task is
-    // slurping a file's entire contents into memory.
-    dat, err := ioutil.ReadFile("/tmp/dat")
-    check(err)
-    fmt.Print(string(dat))
+	// Il task più semplice è la lettura dell'intero
+	// file in memoria
+	dat, err := ioutil.ReadFile("/tmp/dat")
+	check(err)
+	fmt.Print(string(dat))
 
-    // You'll often want more control over how and what
-    // parts of a file are read. For these tasks, start
-    // by `Open`ing a file to obtain an `os.File` value.
-    f, err := os.Open("/tmp/dat")
-    check(err)
+	// Generalmente si vuole più controllo sul modo con
+	// viene letto un file. Per queste funzionalità si
+	// deve iniziare aprendo il file (con `Open`) al fine
```

Comment:
In inglese non ci ha messo `os.` perché se no il "gioco di parole" risultava brutto, ma qua mi sembra più appropriato dire `con os.Open`